backend/spec/helpers/admin/stock_locations_helper_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Spree::Admin::StockLocationsHelper, type: :helper do
+  describe "#admin_stock_location_display_name" do
+    subject { helper.admin_stock_location_display_name(stock_location) }
+
+    context "without admin_name" do
+      let(:stock_location) { create(:stock_location_with_items) }
```

Comment:
Can we set the name explicitly, both here and in the `let` statement below? Right now "NY Warehouse" is a magic string and relies on the factory's defaults—it would be good to make it explicit, for clarity and so that any changes in the factory don't affect the test.

Reply:
It makes a lot of sense. Thanks for the suggestion. Done!